### PR TITLE
options list in packages detail does not apply to project.json

### PR DESCRIPTION
**Pull request by mishra14**
Fixing bug - https://github.com/NuGet/Home/issues/2665

Added warning for solutions which manage their dependencies through project.json

cc @rohit21agrawal @alpaix @joelverhagen @rrelyea

### CODE REVIEW

File: src/NuGet.Clients/PackageManagement.UI/Xamls/OptionsControl.xaml [hunk 1]
```diff
@@ -32,8 +35,18 @@
...
         <RowDefinition Height="auto" />
+        <RowDefinition Height="auto" />
       </Grid.RowDefinitions>
 
+      <Grid Visibility="{Binding IsSolution, Converter={StaticResource BooleanToVisibilityConverter}}" Grid.Row="0" Name = "warningTextGrid" Margin="2,3,2,3">
+        <Grid.ColumnDefinitions>
+          <ColumnDefinition Width="Auto" />
+          <ColumnDefinition Width="*"/>
+        </Grid.ColumnDefinitions>
+        <Image Grid.Row="0" Grid.Column = "0" Name = "WarningTextImage"  Height="16" Width="16" Source="{StaticResource BitmapImage_StatusInformation}"/>
```
Comment (rohit21agrawal) [inline src/NuGet.Clients/PackageManagement.UI/Xamls/OptionsControl.xaml]: if you add something to the start of the grid, then you should update the rest of the elements within the grid with the new positions as well.

In this case, elements in the grid were starting at row number 1, so anything you start in the grid should start with row number 1, and everything below that should have the row number changed.

Comment (rohit21agrawal) [inline src/NuGet.Clients/PackageManagement.UI/Xamls/OptionsControl.xaml]: no need to give a name, until you need to refer to it at another place in XAML or in code-behind.

File: src/NuGet.Clients/PackageManagement.UI/Xamls/OptionsControl.xaml [hunk 2]
```diff
@@ -32,8 +35,18 @@
         <RowDefinition Height="auto" />
         <RowDefinition Height="auto" />
         <RowDefinition Height="auto" />
+        <RowDefinition Height="auto" />
       </Grid.RowDefinitions>
 
+      <Grid Visibility="{Binding IsSolution, Converter={StaticResource BooleanToVisibilityConverter}}" Grid.Row="0" Name = "warningTextGrid" Margin="2,3,2,3">
```
Comment (rohit21agrawal) [inline src/NuGet.Clients/PackageManagement.UI/Xamls/OptionsControl.xaml]: as discussed, moving this to within InstallOptions will improve the logic.

**Comment by alpaix** [comment]
Looks good. Please squash commits before merging into dev (don't use the merge button!).
:shipit:

**Comment by rohit21agrawal** [comment]
:shipit:

**Comment by mishra14** [comment]
Merged : https://github.com/NuGet/NuGet.Client/commit/e34c1928bc4915680e85bdebbfd5f5609deadb1d

**Pull request closed without merging**